backend/curriculum_tracking/api_views.py

```diff
@@ -356,6 +356,8 @@ def request_review(self, request, pk=None):
             card.finish_topic()
         else:
             raise Http404
+        log_creators.log_card_review_requested(card=card, actor_user=request.user) 
+        # comes from here: def log_card_review_requested(card, actor_user) in activity_log_entry_creators.py
```

Comment:
You will need to not commit and push on notes that you leave yourself.  I get what you are doing, I also do that but it does actually confuse others which need to read and understand your code.

Reply:
Noted - the excitement of getting a new test to pass got the better of me, will scrutinise more closely in future ;)